src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1500,8 +1520,11 @@ void GraphOptimizer::FuseConvolutionAndSimpleOperationThroughMaxPool(Graph &grap
 //Disable ACL post-ops in fp16 to avoid performance degradation
 #if defined(OPENVINO_ARCH_ARM64)
         if (parentNode->getOriginalInputPrecisionAtPort(0) == ov::element::f16) {
-            parent++;
-            continue;
+            bool isFirstSupportedPostOp = parentNode->getFusedWith().empty() && is_post_op_supported(childNode);
+            if (!isFirstSupportedPostOp) {
+                parent++;
+                continue;
+            }
         }
 #endif
```

Comment:
1. Lets remove this code block,
2. We just need to check that parentNode->getFusedWith().empty() right before isUnarySupportedAsPostOp call below
3. Check in item 2 should be unser OV_CPU_WITH_ACL macro

Reply:
fixed